src/components/EditHospitalForm/index.js

```diff
@@ -91,43 +87,48 @@ const EditHospitalForm = ({ errors, setErrors, hospital = {}, submit }) => {
           value={hospitalName || ""}
         />
       </FormGroup>
-
-      <FormGroup>
-        <Label htmlFor="hospital-survey-url" className="nhsuk-label--m">
-          Key contact survey URL (optional)
-        </Label>
-        <span className="nhsuk-hint" id="hospital-survey-url-hint">
-          The survey URL will appear at the end of a visit for a key contact.
-        </span>
-        <Input
-          id="hospital-survey-url"
-          type="url"
-          hasError={hasError("hospital-survey-url")}
-          errorMessage={errorMessage("hospital-survey-url")}
-          onChange={(event) => setHospitalSurveyUrl(event.target.value)}
-          name="hospital-survey-url"
-          value={hospitalSurveyUrl || ""}
-        />
-      </FormGroup>
-
-      <FormGroup>
-        <Label htmlFor="hospital-support-url" className="nhsuk-label--m">
-          Key contact support URL (optional)
-        </Label>
-        <span className="nhsuk-hint" id="hospital-support-url-hint">
-          The support URL will appear at the end of a visit for a key contact.
-        </span>
-        <Input
-          id="hospital-support-url"
-          type="url"
-          hasError={hasError("hospital-support-url")}
-          errorMessage={errorMessage("hospital-support-url")}
-          onChange={(event) => setHospitalSupportUrl(event.target.value)}
-          name="hospital-support-url"
-          value={hospitalSupportUrl || ""}
-        />
-      </FormGroup>
-
+      {action === "Add" && (
+        <FormGroup>
+          <Label htmlFor="hospital-code" className="nhsuk-label--m">
+            Create a hospital code
+          </Label>
+          <Input
+            id="hospital-code"
+            type="text"
+            hasError={hasError("hospital-code")}
+            errorMessage={errorMessage("hospital-code")}
+            className="nhsuk-input--width-10"
+            onChange={(event) => setHospitalCode(event.target.value)}
+            name="hospital-code"
+            autoComplete="off"
+            value={hospitalCode || ""}
+          />
+        </FormGroup>
+      )}
+      {action === "Edit" && (
+        <FormGroup>
+          <Label htmlFor="hospital-status" className="nhsuk-label--m">
+            Hospital Status
+          </Label>
+          <Select
+            id="hospital-status"
+            className="nhsuk-input--width-10 nhsuk-u-width-one-half"
+            prompt="Choose a hospital status"
+            options={[
+              { id: 1, name: "active" },
+              { id: 2, name: "disabled" },
+            ]}
+            onChange={(event) => {
+              setHospitalStatus(
```

Comment:
Consider moving this out into its own function

Reply:
I have created a SelectStatus component to handle hospital Status and ward Status (and other future statuses). Please review now. Thanks.